hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -2053,128 +2053,6 @@
-    <value>GRPC</value>
-    <tag>OZONE, SCM, HA, RATIS</tag>
-    <description>Ratis supports different kinds of transports like netty, GRPC,
-      Hadoop RPC etc. This picks one of those for this cluster.
-    </description>
-  </property>
-
-  <property>
-    <name>ozone.scm.ratis.storage.dir</name>
-    <value/>
-    <tag>OZONE, SCM, HA, RATIS, STORAGE</tag>
-    <description>This directory is used for storing SCM's Ratis metadata like
-      logs. If this is not set then default metadata dirs is used. A warning
-      will be logged if this not set. Ideally, this should be mapped to a
-      fast disk like an SSD.
-      If undefined, SCM ratis storage dir will fallback to ozone.metadata.dirs.
-      This fallback approach is not recommended for production environments.
-    </description>
-  </property>
-
-  <property>
-    <name>ozone.scm.ratis.segment.size</name>
-    <value>16KB</value>
-    <tag>OZONE, SCM, HA, RATIS, PERFORMANCE</tag>
-    <description>The size of the raft segment used by Apache Ratis on SCM.
-      (16 KB by default)
-    </description>
-  </property>
-
-  <property>
-    <name>ozone.scm.ratis.segment.preallocated.size</name>
-    <value>16KB</value>
-    <tag>OZONE, SCM, HA, RATIS, PERFORMANCE</tag>
-    <description>The size of the buffer which is preallocated for raft segment
-      used by Apache Ratis on SCM.(16 KB by default)
-    </description>
-  </property>
-
-  <property>
-    <name>ozone.scm.ratis.log.appender.queue.num-elements</name>
-    <value>1024</value>
-    <tag>OZONE, DEBUG, SCM, HA, RATIS</tag>
-    <description>Number of operation pending with Raft's Log Worker.
-    </description>
-  </property>
-  <property>
-    <name>ozone.scm.ratis.log.appender.queue.byte-limit</name>
-    <value>32MB</value>
-    <tag>OZONE, DEBUG, SCM, HA, RATIS</tag>
-    <description>Byte limit for Raft's Log Worker queue.
-    </description>
-  </property>
-  <property>
-    <name>ozone.scm.ratis.log.purge.gap</name>
-    <value>1000000</value>
-    <tag>OZONE, SCM, HA, RATIS</tag>
-    <description>The minimum gap between log indices for Raft server to purge
-      its log segments after taking snapshot.
-    </description>
-  </property>
-  <property>
-    <name>ozone.scm.ratis.server.request.timeout</name>
-    <value>3s</value>
-    <tag>OZONE, SCM, HA, RATIS</tag>
-    <description>The timeout duration for SCM's ratis server request.</description>
-  </property>
-
-  <property>
-    <name>ozone.scm.ratis.server.retry.cache.timeout</name>
-    <value>600000ms</value>
-    <tag>OZONE, SCM, HA, RATIS</tag>
-    <description>Retry Cache entry timeout for SCM's ratis server.</description>
-  </property>
-
-  <property>
-    <name>ozone.scm.ratis.minimum.timeout</name>
-    <value>1s</value>
-    <tag>OZONE, SCM, HA, RATIS</tag>
-    <description>The minimum timeout duration for SCM's Ratis server rpc.
-    </description>
-  </property>
-
-  <property>
-    <name>ozone.scm.ratis.minimum.timeout</name>
```

Comment:
This should not be removed.
And also can we move this to SCMHAConfiguration?

Reply:
And also we can rename this to ozone.scm.ratis.request.timeout

Reply:
Again added the property 'ozone.scm.ratis.minimum.timeout' to ozone-default.xml due to test failure.